source/loop.cpp

```diff
@@ -116,6 +139,11 @@ state_type Loop::CalculateInitialConditions(void)
   double heat = heater->Get_Heating(0.0);
   state_type state;
 
+  if( parameters.use_variable_abundances )
+  {
+      parameters.initial_radiation = true;
```

Comment:
Why is this set to true here and then later set to false in line 174?

Reply:
Ah I see. That makes sense. Could you just add a short comment that just says that?